Project/Project/Pages/PrizeDetails.cshtml.cs

```diff
@@ -17,7 +17,7 @@ public class DetailsModel : PageModel
         public IActionResult OnGet(int? id)
         {
             SearchCompleted = true;
-
+            /* Searching if there are any existing prizes won by Nobel Laureates */
```

Comment:
Only add comments if it is providing value over and above what can be read from the method.